build.gradle.kts

```diff
@@ -27,10 +27,10 @@ repositories {
 }
 
 dependencies {
-    compileOnly("org.jetbrains.kotlin:kotlin-gradle-plugin:1.9.20")
-    compileOnly("com.android.tools.build:gradle:8.0.2")
-    compileOnly("org.jetbrains.intellij.plugins:gradle-intellij-plugin:1.15.0")
-    compileOnly("org.jetbrains.kotlinx:binary-compatibility-validator:0.13.2")
+    compileOnly("org.jetbrains.kotlin:kotlin-gradle-plugin:1.9.23")
+    compileOnly("com.android.tools.build:gradle:8.3.1")
+    compileOnly("org.jetbrains.intellij.plugins:gradle-intellij-plugin:1.17.3")
+    compileOnly("org.jetbrains.kotlinx:binary-compatibility-validator:0.15.0-Beta.2")
```

Comment:
I suggest to wait for version 0.15.0. WDYT?

Reply:
As you wish 🙂 
I will make WIP PRs in all your libraries and mark this one as WIP as well.

BTW, Redwood already adopted it recently, that's why I've decided that it seems to be about stable 🙂  

https://github.com/cashapp/redwood/commit/aac293ce15d713c6dd85abbf3abd058f49eff085#diff-a5040955ccae69f8272686096ca4076bf8c69114111afecfd09c50503df78e4f